include/walkers.h

```diff
@@ -27,7 +27,7 @@ void walk_methods(const T& scope, MethodWalkerFn walker) {
     for (auto vmethod : cls->get_vmethods()) {
       walker(vmethod);
     }
-  };
+  }
```

Comment:
I'd prefer not to take this part of the change, it's extra overhead for my team to review trivial stuff.